tunnel.go

```diff
@@ -73,9 +73,9 @@ type TunnelConnectionResponse struct {
 // other than in the HTTP response unmarshaling. Use `TunnelConfigurationResult`
 // for the correct types.
 type TunnelConfigurationStringifiedConfigResult struct {
```

Comment:
this should be renamed now to better reflect it's use

Reply:
I think it can be removed now (see my latest PR).

I think this is OK. It's not safe since it can be breaking because `TunnelConfigurationStringifiedConfigResult` was an exported type before in the SDK but I doubt anyone was directly referencing those types.